FORM_Entrainement.Altis/functions/CCOM_AT/fn_CCOM_updateLaneAT.sqf

```diff
@@ -0,0 +1,30 @@
+params ["_setting", "_value"];
+
+// Recover the lane identifier :
+private _lane = _setting select [13, 5];
```

Comment:
Is _setting a string ?
This is not very robust

Reply:
yep. Why so ? This is quite a standard way to handle CBA setting changes.

In any case, this will change in the future, see #5

Reply:
Ok if temporary then